materials_data/Al_6061.yaml

```diff
@@ -0,0 +1,134 @@
+          477.5944, 488.7056, 499.8167, 510.9278, 522.0389, 533.1500, 544.2611, 555.3722,
+          566.4833, 577.5944, 588.7056, 599.8167, 610.8056]
+        values: [1.4569, 1.4126, 1.3667, 1.3267, 1.2893, 1.2584, 1.2310, 1.2090, 1.1821,
+          1.1603, 1.1433, 1.1280, 1.1138, 1.1019, 1.0918, 1.0845, 1.0756, 1.0681, 1.0590,
+          1.0496, 1.0392, 1.0288, 1.0166, 1.0055, 0.9947, 0.9824, 0.9686, 0.9543, 0.9405,
+          0.9258, 0.9102, 0.8917, 0.8707, 0.8460, 0.8253, 0.7955, 0.7666, 0.7172, 0.6783,
+          0.6235, 0.5652, 0.5061, 0.4433, 0.3671, 0.3182, 0.2641, 0.2163, 0.1769, 0.1494,
+          0.1298, 0.1138, 0.1028, 0.0934]
+      1000:
+        temperature: [35.5389, 44.2611, 55.3722, 66.4833, 77.5944, 88.7056, 99.8167, 110.9278,
+          122.0389, 133.1500, 144.2611, 155.3722, 166.4833, 177.5944, 188.7056, 199.8167,
+          210.9278, 222.0389, 233.1500, 244.2611, 255.3722, 266.4833, 277.5944, 288.7056,
+          299.8167, 310.9278, 322.0389, 333.1500, 344.2611, 355.3722, 366.4833, 377.5944,
+          388.7056, 399.8167, 410.9278, 422.0389, 433.1500, 444.2611, 453.7778, 466.4833,
+          477.5944, 488.7056, 499.8167, 510.9278, 522.0389, 533.1500, 544.2611, 555.3722,
+          566.4833]
+        values: [1.4493, 1.4126, 1.3667, 1.3267, 1.2893, 1.2584, 1.2310, 1.2081, 1.1821,
+          1.1603, 1.1433, 1.1280, 1.1138, 1.1019, 1.0918, 1.0845, 1.0756, 1.0681, 1.0590,
+          1.0496, 1.0392, 1.0288, 1.0166, 1.0064, 0.9959, 0.9824, 0.9686, 0.9543, 0.9405,
+          0.9258, 0.9102, 0.8917, 0.8707, 0.8485, 0.8221, 0.7994, 0.7663, 0.7135, 0.6319,
+          0.5296, 0.4596, 0.3906, 0.3358, 0.2906, 0.2439, 0.2063, 0.1698, 0.1368, 0.1077]
+      10000:
+        temperature: [36.1389, 44.2611, 55.3722, 66.4833, 77.5944, 88.7056, 99.8167, 110.9278,
+          122.0389, 133.1500, 144.2611, 155.3722, 166.4833, 177.5944, 188.7056, 199.8167,
+          210.9278, 222.0389, 233.1500, 244.2611, 255.3722, 266.4833, 277.5944, 288.7056,
+          299.8167, 310.9278, 322.0389, 333.1500, 344.2611, 355.3722, 366.4833, 377.5944,
+          388.7056, 399.8167, 410.9278, 422.0389, 433.1500, 444.2611, 455.3722, 466.4833,
+          477.5944, 488.7056, 499.8167, 510.9278, 522.0389, 533.1500, 544.2611, 555.3722,
+          566.4833, 577.5944, 588.7056, 599.8167, 610.9278, 622.1500]
+        values: [1.4489, 1.4126, 1.3667, 1.3267, 1.2893, 1.2584, 1.2310, 1.2090, 1.1821,
+          1.1603, 1.1433, 1.1280, 1.1138, 1.1019, 1.0918, 1.0845, 1.0756, 1.0681, 1.0590,
+          1.0496, 1.0392, 1.0288, 1.0166, 1.0059, 0.9925, 0.9824, 0.9686, 0.9543, 0.9405,
+          0.9258, 0.9102, 0.8917, 0.8707, 0.8460, 0.8007, 0.7404, 0.6731, 0.6087, 0.5437,
+          0.4727, 0.4014, 0.3315, 0.2777, 0.2311, 0.1917, 0.1569, 0.1313, 0.1130, 0.0991,
+          0.0868, 0.0772, 0.0683, 0.0641, 0.0594]
+
+forms:
+  'extruded, thickness > 1 inch':
+    conditions:
+      T6:
+        properties:
+          <<: *common_properties
```

Comment:
What does this do? is "<<" a key? Is the asterisk an actual thing?
Never seen before, very nice

Reply:
This is a nifty yaml feature. I added a comment explaining it in https://github.com/mvernacc/material-properties-interchange/pull/1/commits/45263dafa0b6acf491173644e8fb62a6b52a3db6